dbt/adapters/athena/connections.py

```diff
@@ -31,6 +26,11 @@
 
 from dbt.adapters.athena.config import get_boto3_config
 from dbt.adapters.athena.session import get_boto3_session
+from dbt.adapters.base import Credentials
```

Comment:
This should have been the default from the beginning. I am not sure to understand why formatting tools have separated the dbt imports ...

Reply:
re separation: I guess some tools would separate based if the class is defined in this package or the upstream dbt core package. Now that they are seen as one (namespace) package, I guess they get reordered because they are viewed as the same?